pkg/disttask/importinto/proto.go

```diff
@@ -46,6 +46,15 @@ type TaskMeta struct {
 	ChunkMap map[int32][]importer.Chunk
 }
 
+// ImportStepExternalMeta is the external meta of import step.
+type ImportStepExternalMeta struct {
```

Comment:
maybe direct use `ImportStepMeta`, but only set some fields, then there is no need to change structure of subtask meta

Reply:
add `external: “true”` tag